.circleci/config.yml

```diff
@@ -33,6 +33,23 @@ jobs:
     - run:
         command: ./e2e-harness teardown
         when: always
+  e2eTestFrameworkIntegration:
```

Comment:
I would appreciate some new lines between the bigger blocks for visual support of my old eyes.

Reply:
I added a new line above.